charts/cryostat/templates/_helpers.tpl

```diff
@@ -119,7 +119,7 @@ Get or generate a default secret key for object storage.
 Get or generate a default secret key for auth proxy cookies.
 */}}
 {{- define "cryostat.cookieSecret" -}}
-{{- $secret := (lookup "v1" "Secret" .Release.Namespace (printf "%s-cookie-secret" .Release.Name)) -}}
+{{- $secret := (lookup "v1" "Secret" .Release.Namespace ( default (printf "%s-cookie-secret" .Release.Name) .Values.authentication.cookieSecretName )) -}}
```

Comment:
I think these secret templates are only used with generated secrets (i.e.if provided secrets are not used) so we can leave them as they were previously?

That probably avoid misunderstanding that provided secrets are being looked up here.